src/alchemlyb/tests/test_preprocessing.py

```diff
@@ -191,3 +195,54 @@ class TestEquilibriumDetection(TestSlicing, CorrelatedPreprocessors):
     def slicer(self, *args, **kwargs):
         return equilibrium_detection(*args, **kwargs)
 
+class Test_Units():
+    '''Test the preprocessing module.'''
+    @staticmethod
+    @pytest.fixture(scope='class')
+    def dhdl():
+        dataset = load_benzene()
+        dhdl = extract_dHdl(dataset['data']['Coulomb'][0], 310)
+        return dhdl
+
+    def test_kt2kt(self, dhdl):
```

Comment:
The tests for unit conversion should be with the postprocessing tests as these functions live there. Keep the tests localized with the code in the same module as much as possible.If someone implements new functions in postprocessing, they will immediately see the tests in test_postprocessing.py (but not in test_preprocessing.py).